lib/src/utils/crypto/native.dart

```diff
@@ -65,6 +80,219 @@ abstract class Cipher {
       malloc.free(mem);
     }
   }
+
+  Future<IOSink> getOutputIOSink(String outputFilePath) async {
+    final outputFile = await File(outputFilePath).create();
+    return outputFile.openWrite();
+  }
+
+  Future<EncryptedInfo> encryptStream({
+    required Stream<List<int>> inputStream,
+    required File outputFile,
+    required Uint8List key, 
+    required Uint8List initialVector,
+  }) async {
+    final algorithm = EVP_aes_256_ctr();
```

Comment:
IMO, we should support for all available algorithm, but need to verify the `key` length.